docs/source/configure/index.rst

```diff
@@ -146,6 +148,54 @@ Environment-specific parameters
   ``"initial_peers" : [{"address":"127.0.0.1:10001", "public_key":
   "bddd58404d1315e0eb27902c5d7c8eb0602c16238f005773df406bc191308929"}]``
 
+Alternative
+===========
+
+Alternative to using a configuration file is setting up the configuration using environment variables.
+**Configuration file, if present, will have a priority in parameters that are set up in it.**
+
+Here is how it will look like in
```

Comment:
I would suggest to add a bit more formal description on how to set some parameter through env.
first, the lazy approach, let the user set the flag `--verbosity trace` and take the parameter name from the log.
second, the names are formed from the config structure: fixed path labels are taken as is, arrays are indexed starting from `0`, and user-provided dictionary keys are a bit trickier: the key and the value are set in separate vars. everything is uppercased and joined with `_`.